index.html

```diff
@@ -235,7 +235,7 @@ <h2>Glossary</h2>
 
 <p><dfn class="lint-ignore export" data-lt="field-based time format">Field-based formats</dfn>. A time format that divides the date and/or time into separate field values such as year, month, day, hour, minute, second, etc. such as <samp>2016-09-11T06:10:32</samp>. Contrast this with an alternative way to express the same time, <samp>1465621816590</samp>, which is not field-based and is rather hard to read. Field-based times may or may not be tied to either <a href="#def_utc" class="termref">UTC</a> or the local time zone – or they may be indeterminate. Field-based times are also typically tied to a specific calendar (such as the Gregorian calendar). The formats described by the ISO 8601 standard are field-based.</p>
 
-<p class="normative"><dfn class="lint-ignore export">First-strong detection</dfn>. An algorithm that looks for the first strongly-directional character in a string (while ignoring embedded runs of isolated text), and then uses that to guess at the appropriate base direction for the string as a whole. Unicode code points are associated with properties relating to text direction: generally, letters in right-to-left scripts such as Arabic and Hebrew have a strong RTL direction, whereas Latin and Han characters have a strong LTR direction. Other characters, such as punctuation, only have a weak intrinsic directionality, and the actual directionality is determined according to the context in which they are found.</p>
+<p class="normative"><dfn class="lint-ignore export">First-strong detection</dfn>. An algorithm that examines a string from the start, skipping over any characters that are part of isolated runs as defined by the [=Unicode Bidirectional Algorithm=] [[UAX9]], and identifies the strongly-directional character. The base direction of the string is then determined to match the direction of that first strong character. If no strong character is found after examining the entire string, a default base direction (like LTR) is used. Unicode code points are assigned directional properties: letters in right-to-left scripts such as Arabic and Hebrew have strong RTL direction, whereas Latin and Han characters have strong LTR direction. Characters with weak or neutral directional properties, such as punctuation, do not influence the first-strong detection.</p>
```

Comment:
"...identifies the _first_ strongly directional..." (missing an important word)

I would do this differently. You put the setting of the base direction of the string outside the first sentence, but in a glossary definition it should be prioritized. Perhaps:

> **First strong detection** The algorithm used to set the `base direction` of a string or paragraph by finding the first non-isolated strongly-directional character in the text.

... and then go on to explain UAX9, isolation, strong/weak, etc using your or the string-meta text (I believe this is originated in string-meta)

Reply:
Yeah, this is more concise and useful. I adjusted it and divided it into paragraphs.